src/plugins/region_map/server/services/utils/constants.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const DEFAULT_HEADERS = {
+  'Content-Type': 'application/json',
+  Accept: 'application/json',
+  'User-Agent': 'OpenSearch-Dashboards',
+  'osd-xsrf': true,
+};
+export const CLUSTER = {
+  ADMIN: 'admin',
+  GEOSPATIAL: 'opensearch_geospatial',
+  DATA: 'data',
```

Comment:
Where is ADMIN const is used?

Reply:
that was for some test, will remove the unused constants from here.